web/src/test/java/com/epam/rft/atsy/cucumber/settings/positions/PositionsStepDefs.java

```diff
@@ -22,118 +24,228 @@
+  /*
+  @Given("^the positions screen appears$")
   public void the_positions_screen_appears() throws Throwable {
     WebDriverWait wait = new WebDriverWait(getDriver(), 5);
     wait.until(presenceOfElementLocated(By.id("positions")));
     assertThat(getDriver().findElement(By.id("settings")).isDisplayed(), is(true));
     assertThat(getDriver().findElement(By.id("positions")).getTagName(), is("table"));
   }
+  */
 
+  // Anita: ez innen ki lett törölve a feature fájlból mert felesleges
+  /*
   @Given("^the user in on the \"([^\"]*)\" page$")
   public void the_user_in_on_the_page(String arg1) throws Throwable {
     the_options_screen_appears();
   }
+  */
 
-  @Then("^the list filled with positions appears on the page$")
-  public void the_list_filled_with_positions_appears_on_the_page() throws Throwable {
-    int elementCount = getDriver().findElements(By.cssSelector("table#positions tbody tr")).size();
-    assertThat(elementCount, greaterThan(0));
+  @Given("^the list is saved$")
+  public void the_list_saved() throws Throwable {
+    positionsListContents =
+            getDriver().findElement(By.cssSelector("table#positions tbody")).getText();
   }
 
-  @When("^the Mentés button clicked$")
+  @When("^the user clicks on Mentés button$")
   public void the_mentes_button_clicked() throws Throwable {
     getDriver().findElement(By.cssSelector("#position-form .btn-success")).click();
   }
 
-  @When("^the Mégsem button clicked$")
+  @When("^the user clicks on Mégsem button$")
   public void the_megsem_button_clicked() throws Throwable {
     getDriver().findElement(By.cssSelector("#position-form .btn-danger")).click();
   }
 
-  @Then("^the title field cleared$")
-  public void the_title_field_cleared() throws Throwable {
-    String nameValue = getDriver().findElement(By.id("position_name")).getAttribute("value");
-    assertThat(nameValue, isEmptyOrNullString());
-  }
-
-  @And("^user enters \"([^\"]*)\" into the title$")
+  @When("^the user enters (.*) into the title$")
   public void user_enters_into_the_title(String title) throws Throwable {
     getDriver().findElement(By.id("position_name")).clear();
     getDriver().findElement(By.id("position_name")).sendKeys(title);
   }
 
-  @Then("^the new position called (.*) appears in the list of positions$")
+  @When("^the user clicks on Edit button on a position$")
+  public void the_Edit_button_clicked_on_a_position() throws Throwable {
+    List<WebElement> rows = getDriver().findElements(By.cssSelector("table#positions tbody tr"));
+    WebElement firstRow = rows.get(0);
+    positionName = firstRow.getText();
+    firstRow.findElement(By.cssSelector("a.edit")).click();
+  }
+
+  @When("^the user clicks on (.*) tab link$")
+  public void theUserClicksOnTabLink(String tab) throws Throwable {
+      switch (tab) {
+          case "Pozíciók":
+              driverProvider.getDriver().findElement(By.id("positions_link")).click();
```

Comment:
`driverProvider.getDriver()` and `getDriver()` is the same in this file.

Please replace the current call to `getDriver()`.